### ISSUE TITLE
Text in certain dialogs is mashed/overlapping during display

### ISSUE THREAD

**Issue by leejeffr** [robotframework/RIDE#2431]
The last letter of the current word and first letter of next word in certain dialogs is running together when rendered to display. 

Environment: RIDE v2.0b2.dev4 running on Python 3.9.5. 
Running on windows 10 x64
Resolution: 1920 x 1080

![image](https://user-images.githubusercontent.com/5108841/152258289-053a9da0-744e-42f0-8463-aa4cc4f9f883.png)

![image](https://user-images.githubusercontent.com/5108841/152258341-f7cf2386-b253-4e7c-a1f6-099ad62a762f.png)

**Comment by HelioGuilherme66** [comment]
@leejeffr This kind of problem, looks like is not from RIDE but from other related software. Could be wxPython problem, or Windows, or the Font type selected, or even hardware related problem, like graphic card drivers.

I don't see much I can do. Only recommend for you to try to change the Font Type, and also to report what is the wxPython version.

**Comment by leejeffr** [comment]
pip show wxpython
Name: wxPython
Version: 4.1.1
Summary: Cross platform GUI toolkit for Python, "Phoenix" version
Home-page: http://wxPython.org/
Author: Robin Dunn
Author-email: robin@alldunn.com
License: wxWindows Library License (https://opensource.org/licenses/wxwindows.php)
Location: c:\users\jeff.lee\appdata\local\programs\python\python39-32\lib\site-packages
Requires: numpy, pillow, six
Required-by: robotframework-ride

**Comment by leejeffr** [comment]
when I first launch ride it looks fine. Notice the file menu is not cut off. 

![image](https://user-images.githubusercontent.com/5108841/152452562-781f310c-f1b8-4b71-84ca-4bee1ce45a07.png)

then I browse around and ride automatically resizes the window from full screen to a smaller size. 

![image](https://user-images.githubusercontent.com/5108841/152452751-2559b78b-9b26-4622-b3df-ceaf9a347b43.png)

**Comment by leejeffr** [comment]
I think I figured it out. My primary monitor was set to 125% under scale and layout. My secondary monitors were set to 100%. When I moved the app from one screen to another it would cause the overlapping text. Changing the primary monitort to match the others at 100% fixed the render/overlapping display issue with the text. 

You may close the issue. 

![image](https://user-images.githubusercontent.com/5108841/152454061-76a3f051-2eb1-448d-b0b6-bd1ea04f7d3f.png)

**Comment by HelioGuilherme66** [comment]
Yes, this is something out of scope of RIDE handling. An entry to the FAQ would be nice to have.
